Sources/Contentful/Query.swift

```diff
@@ -689,44 +677,22 @@ public class Query: ChainableQuery {
         self.parameters = [String: String]()
     }
 
-    /// Initialize a new query specifying the `content_type` parameter to narrow the results to 
-    /// entries that have that content type identifier. 
-    /// See: <https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters>
-    public convenience init(onContentTypeFor id: ContentTypeId) {
-        self.init()
-        self.on(contentTypeFor: id)
-    }
-
-    /** 
-     Append the `content_type` parameter to narrow the results to entries that have that content type identifier.
-     See: <https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/search-parameters>
-     - Parameter id: the identifier of the content type which the query will be performed on.
-     - Returns: A reference to the receiving query to enable chaining.
-     */
-    @discardableResult public func on(contentTypeFor id: ContentTypeId) -> Query {
-        self.parameters[QueryParameter.contentType] = id
-        return self
-    }
-
     // MARK: Query.Private
 
-    fileprivate init(parameters: [String: String] = [:], locale: String? = nil) {
+    internal init(parameters: [String: String] = [:]) {
         self.parameters = parameters
-
-        if let locale = locale {
-            self.parameters[QueryParameter.locale] = locale
-        }
     }
 
-    fileprivate class func validate(selectedKeyPaths: [String]) throws {
+
+    fileprivate static func validate(selectedKeyPaths: [String]) throws {
         for fieldKeyPath in selectedKeyPaths {
             guard fieldKeyPath.isValidSelection() else {
                 throw QueryError.invalidSelection(fieldKeyPath: fieldKeyPath)
             }
         }
     }
 
-    fileprivate class func addSysIfNeeded(to selectedFieldNames: [String]) -> [String] {
+    fileprivate static func addSysIfNeeded(to selectedFieldNames: [String]) -> [String] {
         var completeSelections = selectedFieldNames
```

Comment:
static just means that if i change the type from class to struct or something like that, that it will still compile ;-) `class func` and `static func` are identical except `class func` only works for classes.

Reply:
:+1: 

I was refering to this line, sadly the highlight did not quite work in our favor: `var completeSelections = selectedFieldNames` I am not sure why that is here ... ;)